app/src/main/java/me/hufman/androidautoidrive/carapp/notifications/views/DetailsView.kt

```diff
@@ -229,10 +231,28 @@ class DetailsView(val state: RHMIState, val phoneAppResources: PhoneAppResources
 			clearButton.setEnabled(false)
 		}
 
+		val hasReadoutAction = !notificationSettings.shouldReadoutNotificationDetails()
+		val readoutButton = buttons[1]
+		if (hasReadoutAction) {
+			readoutButton.apply {
+				setEnabled(true)
+				getTooltipModel()?.asRaDataModel()?.value = L.NOTIFICATION_READOUT_ACTION
```

Comment:
It would be fun to change the icon for this button:
`it.getImageModel()?.asImageIdModel()?.imageId = 154`

It would mean bringing the normal `it.getImageModel()?.asImageIdModel()?.imageId = 158` down into the main actions loop below, but the idempotent rhmi wrapper means it is usually free.

Reply:
good idea :-)

Reply:
done, image-ids are now all assigned dynamically in redraw(). There is no point in pre-set any as none of them is really predefined (e.g. the notification might not be clearable but being read automatically)